content/applications/finance/accounting/others/adviser/budget.rst

```diff
@@ -1,157 +1,101 @@
+:guilabel:`New` to create a new budget for *Smith & Co* project.
 
-.. image:: budget/budget07.png
-   :align: center
-
-Select the Budgetary Position related to the Budget Line. In other
-words, select the position that points to the accounts you want to
-budget. In this case we will start with our 700 maximum charge target.
-Select the "Cost" Budgetary Position and specify the Planned Amount.
-As we are recording a cost, we need to specify a **negative amount**.
-Finally, select the corresponding analytic account.
+First, fill in your :guilabel:`Budget Name`. Then, select the :guilabel:`Period` wherein the budget
+is applicable. Next, add the :guilabel:`Budgetary Position` you want to track, define the related
+:guilabel:`Analytic Account`, and add the :guilabel:`Planned Amount`.
 
-.. image:: budget/budget08.png
+.. image:: budget/define-the-budget.png
    :align: center
+   :alt: budget lines display
 
-Click on **Save & new** to input the revenue budget. The Budgetary
-Position is Revenue and the Planned Amount is 1000. Save and close
-
-You'll need to **Confirm** and **Approve** the budget.
+.. Note::
+   When recording a planned amount related to expenses, the amount must be negative.
 
 Check your budget
-=================
+-----------------
 
-You can check your budget at any time. To see the evolution, let's book
-some Invoices and Vendors Bills.
+Go to :menuselection:`Accounting --> Management: Budgets` and find the *Smith & Co* Project to see
+how it evolves according to the expenses or income for the related analytic account.
 
-.. tip::
-   If you use analytical accounts remember that you need to specify the account in the invoice
-   and/or purchase line.
+The :guilabel:`Practical Amount` evolves when a new journal entry related to an accounts from your
+budgetary position and your analytic account is created.
 
-.. seealso::
-   For more information about booking invoices and purchase orders see
-   :doc:`../../receivables/customer_invoices/overview`
-
-Go back in the budget list and find the Smith Project.
-
-Via the analytical account, Odoo can account the invoice lines and
-purchase lines booked in the accounts and will display them in the
-**Practical Amount** column.
-
-.. image:: budget/budget09.png
-   :align: center
-
-.. note::
-   The theoretical amount represents the amount of money you theoretically could
-   have spend / should have received in function of the date. When your budget
-   is 1200 for 12 months (january to december), and today is 31 of january, the
-   theoretical amount will be 100, since this is the actual amount that could
-   have been realised.
+The :guilabel:`Theoretical Amount` represents the amount of money you theoretically could have spent
+or should have received based on the date. For example, if your budget is 1200 for 12 months
+(January to December), and today is 31 of January, the theoretical amount will be 100, since this is
+the actual amount that could have been made.
```

Comment:
here's what grammarly suggest: 
For example, suppose your budget is 1200 for 12 months (January to December), and today is 31 of January. In that case, the theoretical amount will be 100 since this is the actual amount that could have been made.